docs/misc/12_support_and_sla/index.mdx

```diff
@@ -0,0 +1,63 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Support & SLA
+
+Windmill is committed to delivering comprehensive support and maintaining superior service levels. Our Support & Service Level Agreements (SLAs) are specifically designed to provide timely assistance and address any issues or concerns that may arise during your usage of the Windmill platform.
+
+## Technical Support
+
+Our knowledgeable support team is available 24/7 to assist you with any technical inquiries, troubleshooting, or questions you may have. We are committed to providing timely responses and resolutions to ensure a seamless experience with Windmill.
+
+## Issue Prioritization
+
+To effectively handle and prioritize issues, we categorize them into different priority levels based on their impact and urgency. This enables us to allocate resources efficiently and ensure that critical issues receive immediate attention.
+
+- P0 (Priority 0): Reserved for blocking and urgent issues that require immediate resolution.
+- P1 (Priority 1): High-priority issues that significantly impact functionality or performance.
+- P2 (Priority 2): Medium-priority issues that have a moderate impact on functionality.
+- P3 (Priority 3): Low-priority issues that have minimal impact on functionality or usability.
+
```

Comment:
should mention here that we track issues through private issues boards and bug trackers such as github ones.